targets/csharp/source/Plugins/CloudScript/source/PlayFabFunctionContexts.cs

```diff
@@ -108,33 +106,30 @@ public class FunctionPlayerPlayStreamContext<TFunctionArgument>
         /// </summary>
         /// <param name="request">The request incoming to an Azure Function from the PlayFab server</param>
         /// <returns>A new populated <c>FunctionPlayerPlayStreamContext</c> instance</returns>
-        public static async Task<FunctionPlayerPlayStreamContext<TFunctionArgument>> Create(HttpRequestMessage request)
+        public static async Task<FunctionPlayerPlayStreamContext<TFunctionArgument>> Create(HttpRequest request)
         {
-            using (var content = request.Content)
+            string body = await request.ReadBodyAsStringAsync();
+            var contextInternal = PlayFabSimpleJson.DeserializeObject<FunctionPlayerPlayStreamContextInternal>(body);
+            var settings = new PlayFabApiSettings
             {
-                var body = await content.ReadAsStringAsync();
-                var contextInternal = PlayFabSimpleJson.DeserializeObject<FunctionPlayerPlayStreamContextInternal>(body);
-                var settings = new PlayFabApiSettings
-                {
-                    TitleId = contextInternal.TitleAuthenticationContext.Id,
-                    DeveloperSecretKey = Environment.GetEnvironmentVariable("PLAYFAB_DEV_SECRET_KEY", EnvironmentVariableTarget.Process)
-                };
-
-                var authContext = new PlayFabAuthenticationContext
-                {
-                    EntityToken = contextInternal.TitleAuthenticationContext.EntityToken
-                };
-
-                return new FunctionPlayerPlayStreamContext<TFunctionArgument>()
-                {
-                    ApiSettings = settings,
-                    AuthenticationContext = authContext,
-                    PlayStreamEventEnvelope = contextInternal.PlayStreamEventEnvelope,
-                    PlayerProfile = contextInternal.PlayerProfile,
-                    FunctionArgument = contextInternal.FunctionArgument,
-                    CurrentPlayerId = contextInternal.PlayerProfile.PlayerId
-                };
-            }
+                TitleId = contextInternal.TitleAuthenticationContext.Id,
+                DeveloperSecretKey = Environment.GetEnvironmentVariable("PLAYFAB_DEV_SECRET_KEY", EnvironmentVariableTarget.Process)
+            };
+
+            var authContext = new PlayFabAuthenticationContext
+            {
+                EntityToken = contextInternal.TitleAuthenticationContext.EntityToken
+            };
+
+            return new FunctionPlayerPlayStreamContext<TFunctionArgument>()
+            {
+                ApiSettings = settings,
+                AuthenticationContext = authContext,
+                PlayStreamEventEnvelope = contextInternal.PlayStreamEventEnvelope,
+                PlayerProfile = contextInternal.PlayerProfile,
+                FunctionArgument = contextInternal.FunctionArgument,
+                CurrentPlayerId = contextInternal.PlayerProfile.PlayerId
+            };
```

Comment:
Can you help me understand why there's two seemingly identical copy paste copies of this code?